docker/Dockerfile

```diff
@@ -11,4 +11,6 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Requirement, note that v2 or later runs into issues with gem install due to Ruby 2 req
-RUN gem install net-sftp -v 1.1.1    
\ No newline at end of file
+RUN gem install net-sftp -v 1.1.1
+
+RUN chsh -s /bin/bash root
```

Comment:
Yeah, that probably does nothing - it was a lurking remain from experiments to deal with build failures that turned out to be due to a bad kernel version on one of ci.jenkins-ci.org's slaves. I'll remove it.